build.sh

```diff
@@ -20,9 +20,9 @@ function build {
 
 
 
-  if ! go version | egrep -q 'go(1[.]9|1[.]1[0-9])' ; then
-    echo "go version is too low. Must use 1.9 or above"
-    exit 1
+  if ! go version | egrep -q 'go(1\.1[234])' ; then
+    echo "go version must be 1.12 or above"
+    ok=1
```

Comment:
Is this used anywhere? I didn't see it in the scripts.

Reply:
nice catch! fixed.